tools/topology/topology2/platform/intel/speaker-echo-ref.conf

```diff
@@ -17,6 +17,7 @@ Object.Pipeline {
 	dai-copier-be [
 		{
 			index	$ECHO_REF_DAI_PIPELINE_ID
+			direction	"capture"
```

Comment:
Why didn't we need `direction` before?

Reply:
its never used before. direction is only meant for IPC3. But now I need it for the name